packages/matchbox/src/components/Button/styles.js

```diff
@@ -0,0 +1,136 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export const base = (props) => `
+    display: inline-block;
+    border-radius: ${tokens.borderRadius_100};
+    user-select: none;
+    font-weight: ${tokens.fontWeight_medium};
+    text-decoration: none;
+    white-space: nowrap;
+    vertical-align: middle;
+    transition-property: background, color, border, outline;
+    transition-duration: .15s;
```

Comment:
Should transition duration be a design token? Same with a `transition-timing-function`. I usually feel like `ease-in-out` looks a little nicer than the default (which I believe is `linear`).

Reply:
Yeah definitely should have a set of tokens for animation – will try to find time to discuss with UX team